install_opi5.sh

```diff
@@ -0,0 +1,21 @@
+apt-get update
+apt-get upgrade -y
+
+cd /tmp 
+wget https://git.io/JJrEP -O install.sh
+chmod +x install.sh
+
+sed -i 's/# AllowCPUs=4-7/AllowCPUs=4-7/g' install.sh
```

Comment:
I've done some research and I think this is supposed to be AllowedCPUs. See https://manpages.ubuntu.com/manpages/jammy/en/man5/systemd.resource-control.5.html